src/index.ts

```diff
@@ -84,18 +86,20 @@ export async function newEventStore(pool: Pool, emit: Emitter): Promise<EventSto
         .digest('hex');
 
       const latestSnapshot = await pool.query(aggregateQuery, [id])
-        .then((aggResult) => Option.of(aggResult.rows[0]));
+        .then((aggResult) => Option.of(aggResult.rows[0] ? aggResult.rows[0].data : null));
```

Comment:
The reason for [this comment](https://github.com/repositive/event-store/pull/5/files#r206524922) stems from here; `latestSnapshot` is _always_ an option, but what we pass into the aggregate creation through the third `accumulator` arg could be anything, which makes the reducers have different types on their first and subsequent calls.